src/components/Login.js

```diff
@@ -1,23 +1,44 @@
 import React, { Component } from 'react'
+import { connect } from 'react-redux'
 import styled from 'styled-components'
-import LoginWithSlackButton from "./LoginWithSlackButton";
+import LoginWithSlackButton from './LoginWithSlackButton'
+import { loginInit } from '../store/actions/auth.actions'
 
 class Login extends Component {
     render () {
+        const { pending } = this.props;
         return (
             <Container>
                 <h4>Kto tam?</h4>
+
+                <h5>HIPOPOTAM</h5>
                 <LoginWithSlackButton/>
+                {/*{pending && <div>Trwa pukanie...</div>}*/}
```

Comment:
no pending?

Reply:
Yes, because we are redirected between app and slack, so pending will never be visible :<